cgmes/cgmes-conversion/src/test/resources/logback-test.xml

```diff
@@ -14,7 +14,7 @@
             <pattern>%-5p %-20C{1} | %m%n</pattern>
         </encoder>
     </appender>
-    <root level="info">
+    <root level="error">
```

Comment:
Only impact on raising the error level for test is that we may reduce the coverage of some code.